fhir-model/src/main/java/com/ibm/fhir/model/type/DateTime.java

```diff
@@ -29,10 +29,13 @@
  * A date, date-time or partial date (e.g. just year or year + month). If hours and minutes are specified, a time zone 
  * SHALL be populated. The format is a union of the schema types gYear, gYearMonth, date and dateTime. Seconds must be 
  * provided due to schema type constraints but may be zero-filled and may be ignored. Dates SHALL be valid dates.
+ * If seconds are specified, fractions of seconds may be specified up to nanosecond precision (9 digits). However, any 
```

Comment:
pretty minor, but I'd like to see a `<p>` tag here.  otherwise the javadoc gets consensed into one long paragraph and its not as clear whats from the spec vs from us

Reply:
Fixed.